CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - New style component `header-filesize` to show size of the displayed file in the header. See #1988 (@mdibaiee)
 - Use underline for line highlighting on ANSI, see #1730 (@mdibaiee)
+- Mapped clang-format config file (.clang-format) to TOML syntax (@TruncatedDinosour)
```

Comment:
in the changelog you wrote TOML, in the syntax mapping you wrote YAML :sweat_smile:

Reply:
> in the changelog you wrote TOML, in the syntax mapping you wrote YAML sweat_smile

whoops